third_party/accessibility/ax/platform/ax_fragment_root_win.cc

```diff
@@ -289,7 +289,7 @@ class AXFragmentRootMapWin {
 
 AXFragmentRootWin::AXFragmentRootWin(gfx::AcceleratedWidget widget,
                                      AXFragmentRootDelegateWin* delegate)
-    : widget_(widget), delegate_(delegate) {
+    : widget_(widget), delegate_(delegate), alert_node_(nullptr) {
```

Comment:
What's the benefit of lazily creating the alert node? Could we create an alert node every time we create the root?

Reply:
The alert node, delegate, and fragment root refer to each other, so one pointer or more will be set after creation.

Reply:
Right... but the creation of the fragment root node and alert node is decoupled: a fragment root is created only when I attach a screen reader, and, an alert node is created only when the framework sends an alert. Why is that? Why doesn't creating a fragment root also create the alert node? Do we need to create an alert node if there's no screen reader?

If we do need to create an alert node even without a screen reader, perhaps that should also create a fragment root? That would address concerns like: https://github.com/flutter/engine/pull/38745#discussion_r1066448031